api/pkg/s3/listbuckets.go

```diff
@@ -38,7 +38,13 @@ func parseListBuckets(list *s3.ListBucketsResponse) []byte {
 	buckets := []model.Bucket{}
 	for _, value := range list.Buckets {
 		ctime := time.Unix(value.CreateTime, 0).Format(time.RFC3339)
-		bucket := model.Bucket{Name: value.Name, CreateTime: ctime, LocationConstraint: value.DefaultLocation}
+		versionOpts := model.VersioningConfiguration{}
```

Comment:
Versioning is not get from database when list buckets, so it's invaluable to get versioning here.